package Customexceptions;

public class Nestedtrycatch {
	public static void main(String[] args) {
		try {
			
			try {
				int arr[] = {10,20,30};
				System.out.println(arr[5]);
			}
			catch(ArrayIndexOutOfBoundsException e) {
				System.out.println("array index exception handled");
			}
			
			int a = 10;
			int b = 0;
			
			int result = a/b;
			System.out.println(result);
		}
		catch(ArithmeticException e) {
			System.out.println("Arthimetic exception handled");
		}
	}
}
